ComponentLibWithXmlLinkerConfig/ComponentLibWithXmlLinkerConfig.xml

```diff
@@ -1 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8" ?> 
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<linker>
+  <assembly fullname="System.Core" preserve="all"/>
```

Comment:
Do you have any estimate of what additional compressed payload size this entails? If it's in the < 200kb range it might be satisfactory as is.

If it adds a much larger payload, we might want to spend some time trying to make this preservation list more minimal.

Reply:
The size jumps from 1.83 MiB to 2.13 MiB, so it's right on the threshold of tolerability in this case.

Reply:
OK, I fined tuned the config to just preserve `System.Linq.Queryable`. This still allows the app to work, and the size only jumps from 1.83 MiB to 1.84 MiB.